services/web/client/source/class/osparc/utils/Classifiers.js

```diff
@@ -25,47 +25,72 @@ qx.Class.define("osparc.utils.Classifiers", {
 
   statics: {
     getClassifiersAsTree: function() {
-      return osparc.data.Resources.get("classifiers")
-        .then(({classifiers}) => {
-          // Converts the classifiers to a tree-shaped object
-          const buildTree = (classifierId, currentPath, currentNode) => {
-            const pathName = currentPath.shift();
-            if (currentPath.length) {
-              currentNode[pathName] = currentNode[pathName] || {};
-              buildTree(classifierId, currentPath, currentNode[pathName]);
-            } else {
-              currentNode[pathName] = classifiers[classifierId];
+      return new Promise((resolve, reject) => {
+        const rootData = {
+          label: "root",
+          children: []
+        };
+        osparc.store.Store.getInstance().getGroupsOrganizations()
+          .then(orgs => {
+            if (orgs.length === 0) {
+              reject();
+              return;
             }
-          };
-          // Converts the treefied classifiers to a Qooxdoo's VirtualTree friendly format
-          const virtualTree = currentNode => Object.entries(currentNode).map(([key, value]) => {
-            if (value.classifier) {
-              return {
-                label: value["display_name"],
-                data: value
+            const classifierPromises = [];
+            orgs.forEach(org => {
+              const params = {
+                url: {
+                  "gid": org["gid"]
+                }
               };
-            }
-            return {
-              label: key.charAt(0).toUpperCase() + key.slice(1),
-              children: virtualTree(currentNode[key])
-            };
+              classifierPromises.push(osparc.data.Resources.get("classifiers", params));
+            });
+            Promise.all(classifierPromises)
+              .then(classifierss => {
```

Comment:
MINOR: classifiersS<-- this is on purpose or a typo?

Reply:
On purpose. classifierss: list of classifiers